types/chart.js/index.d.ts

```diff
@@ -33,7 +33,7 @@ declare class Chart {
     update: ({duration, lazy, easing}?: Chart.ChartUpdateProps) => {};
     render: ({duration, lazy, easing}?: Chart.ChartRenderProps) => {};
     stop: () => this;
-    resize: () => this;
+    resize: () => void;
```

Comment:
Why did you ignore the boolean param the function takes as input? https://github.com/chartjs/Chart.js/blob/cbace1cfe2c1b70ac26d1ee798eb5841fcd978c5/src/core/core.controller.js#L264